src/index.html

```diff
@@ -2,16 +2,176 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname-input">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname-input"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="name-input">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name-input"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="age-input">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age-input"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="form-field">
+          <label for="birth-date-input">Full date of birth: </label>
+          <input
+            type="date"
+            name="birth-date"
+            id="birth-date-input"
+          >
+        </div>
+        <div class="form-field">
+          <label for="agreement-input">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement-input"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label>E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>Password:
```

Comment:
```suggestion
          <label>
            Password:
```

This text better to move to the next line to improve readibility